main.py

```diff
@@ -1,12 +1,168 @@
+    #     print("You must select a customer before altering")
+    #     print("Let's select a customer!")
+    #     choice = "10"
+
+    # if choice in ['2','3','5'] and video.selected_video == None:
+    #     print("You must select a video before altering")
+    #     print("Let's select a video!")
+    #     choice = "5"
+
+
+    # if choice in valid_choices:
+    #     print("")
+    #     print("")
+        return choice
+
+def get_video_info():
+    title = input("Please enter title: ")
+    release_date = input("Please enter release_date: ")
+    inventory = input("Please enter total inventory: ")
+    available_inventory = input("Please enter available inventory: ")
+    return title, release_date, inventory, available_inventory
+
+def run_cli(play=True):
+
+    customer = Customer(url="https://retro-video-store-api.herokuapp.com")
+    video = Video(url="https://retro-video-store-api.herokuapp.com")
+
+    options = list_options()
+
+    while play==True:
+        choice = make_choice(options)
+
+        if choice == "1":
+            title = input("Please enter title: ")
+            release_date = input('Enter a date in YYYY-MM-DD format:  ')
+            total_inventory = input("Please enter total inventory: ")
+            print(f"Video profile '{title}' has been created.")   
+            response = video.create_video(title=title, release_date=release_date, total_inventory=int(total_inventory))
+
+        elif choice == "2":
+            video_id = input("Please enter Video ID:  ")
+            title = input("Please enter title: ")
+            release_date = input('Enter a date in YYYY-MM-DD format:  ')
+            total_inventory = input("Please enter total inventory: ")
+            print(f"Video profile '{title}' has been updated.")   
+            response = video.update_video(video_id=video_id, title=title, release_date=release_date, total_inventory=int(total_inventory))
+
+        elif choice == "3":
+            video_id = input("Please enter Video id for deletion: ")
+            response = video.delete_video(video_id=video_id)
+            print(f"Video id '{video_id}' has been deleted")
+
+        elif choice == "4":
+            for video in video.all_videos(): 
+                print(video)
+
+        elif choice == "5":
+            video_id = input("Please enter Reel id.")
+            if video_id.isnumeric():
+                video_id = int(video_id)
+                info = video.get_video(video_id=video_id)
+                print(info)
+        
+        elif choice == "6":
+            customer_name = input("Please enter customer name:  ")
+            postal_code = input("Please enter customer postal code:  ")
+            phone_number = input("Please enter customer phone number:  ")
+            respone = customer.create_customer(customer_name=customer_name, postal_code=postal_code, phone_number=phone_number)
+            print(f"Customer profile '{customer_name}' has been created.")
```

Comment:
If the API responds with an error code, this print statement will still happen. See comments on the create_customer method for more details